src/helper/validation.ts

```diff
@@ -219,6 +219,10 @@ const validate_preview_args = (user_args: string[]) => {
         logger.error(`Invalid Or Missing Argument: store_email`)
     }
 
+    if (user_args[3] && user_args[3] === '--theme_name' && user_args[4]) {
```

Comment:
I suggest renaming the argument to `--theme_code` to be more relevant, since we are actually sending the theme code not the name.